File: edu/javascript/explore.js
  const Twilio = require('twilio');

  const Client = new Twilio ("AC806b40d5a1074f6170d14a60d7991992","0d0410504731137b44b56ad2cf154054");

  Client.messages.list()
    .then(messages =>
        console.log(`the most recent message is ${messages[0].body}`)
        ).catch(err => console.error(err));; 

        console.log(`Gathering your message`)

File: edu/python/explore.py
from twilio.rest import Client

client = Client("AC806b40d5a1074f6170d14a60d7991992","0d0410504731137b44b56ad2cf154054")

#for msg in client.messages.list():
#    print(msg.body)

msg = client.messages.create(
    to="+4528141529",
    from_="+12543292339",
    body="helo mister Glen",
)

print(f"Created a new messesage: {msg.sid}")